### DISCUSSION TITLE
Is there an MSBuild variable to add a triplet overlay path like VCPK_OVERLAY_TRIPLETS?

### DISCUSSION THREAD

**Discussion by meastp** [microsoft/vcpkg#19051] [Ideas]
I have looked in the documentation, but couldn't find an MSBuild variable/property to add a triplet overlay path.

Does this exist? If not, this is a feature request... :)

In combination with the manifests feature it would be useful to keep custom triplets in the project source repo, and not have to modify global environment variables like VCPKG_OVERLAY_TRIPLETS but instead keep the configuration in an MSBuild .props file.

```xml
  <!-- Custom project triplets for vcpkg -->
  <PropertyGroup>
    <VcpkgTriplet Condition="'$(Platform)'=='Win32'">x86-windows-$(PlatformToolset)</VcpkgTriplet>
    <VcpkgTriplet Condition="'$(Platform)'=='x64'">x64-windows-$(PlatformToolset)</VcpkgTriplet>
    <VcpkgOverlayTriplets>$(MSBuildThisFileDirectory)/custom_triplets</VcpkgOverlayTriplets>
  </PropertyGroup>
```

example folder structure in project root:
```c
src
vcpkg_installed
custom_triplets // contains triplet files
vcpkg.json
project.vcxproj
```

Building in Visual Studio would then work automatically with MSBuild.

**Comment by meastp** [comment]
I solved this using 

```xml
<VcpkgAdditionalInstallOptions>--overlay-triplets=$(VcpkgManifestRoot)\triplets $(VcpkgAdditionalInstallOptions)</VcpkgAdditionalInstallOptions>
```

So there is no urgent need for a specialized property for this.

**Comment by JackBoosY** [comment]
Please keep this issue open until we fix it by adding a feature.
